packages/riverpod_generator/test/dependencies_test.dart

```diff
@@ -106,6 +106,18 @@ void main() {
       transitiveDependenciesProvider.dependencies,
       same(transitiveDependenciesProvider.dependencies),
     );
+    expect(
+      smallTransitiveDependencyCountProvider.dependencies,
+      same(smallTransitiveDependencyCountProvider.dependencies),
```

Comment:
That's not what I meant

Do `same(const <ProviderOrFamily>[])`

Reply:
I wrote it this way because it's a test to check if dependencies are cached in this part.
[cef6d9c](https://github.com/rrousselGit/riverpod/pull/2310/commits/cef6d9c570bd0527e134334a584e7c1981b05c31)

The tests for "Supports specifying dependencies" and "Generates transitive dependencies" are written as you pointed out.
[87a14ca](https://github.com/rrousselGit/riverpod/pull/2310/commits/87a14ca70888addd0de4739bdeb4903747d77ac2)

Reply:
@rrousselGit I have responded to the comment you provided, and I would appreciate it if you could take a moment to review my response